beacon-chain/core/helpers/committee.go

```diff
@@ -174,7 +174,7 @@ type CommitteeAssignmentContainer struct {
 func CommitteeAssignments(
 	state iface.BeaconState,
 	epoch types.Epoch,
-) (map[types.ValidatorIndex]*CommitteeAssignmentContainer, map[types.ValidatorIndex][]types.Slot, error) {
+) (assignments map[types.ValidatorIndex]*CommitteeAssignmentContainer, proposals map[types.ValidatorIndex][]types.Slot, err error) {
```

Comment:
unless there is a good reason, remove the named return values here. They aren't used anywhere in this function.

Reply:
The reason is to understand what the second return value (`map[types.ValidatorIndex][]types.Slot`) is for. I can make some changes to the code and assign values to these variables.

Reply:
Named return values are rarely used because they make understanding the control flow of a function/method more difficult. Unless there is a strong reason we shouldn't start using named values here